### ISSUE TITLE
Failing to export/import messages. "0 SMS(s) and 0 MMS(s) imported"

### ISSUE THREAD

**Issue by Savasdotexe** [tmo1/sms-ie#202]
After trying various combinations of settings, I'm at a loss why my SMS messages are not importing.

When exporting, I do get a message of 1900 sms and 56 mms exported in 10 seconds. The zip file size is 846KB, I can see some media in the data folder and the messages.ndjson file is 1531 KB so I am guessing it is exporting correctly, just not importing. Any suggestions or fixes?

Here's some screenshots.

Exporting from: Samsung A52 Android 14 (one ui 6.1)
Importing to: Samsung A21s android 10 (one ui 2.1)

![Screenshot_20240728-130941_SMS Import  Export](https://github.com/user-attachments/assets/c4481603-5b40-4ee3-8721-5c52cf479899)

![Screenshot_20240728-130949_SMS Import  Export](https://github.com/user-attachments/assets/d1d1b47e-4b78-430f-8da5-5ce3dcbe3e7e)

![Screenshot_20240728-130958_SMS Import  Export](https://github.com/user-attachments/assets/079e4bc5-d644-48d0-88af-c1dc7709d7d8)

**Comment by tmo1** [comment]
> I'm at a loss why my SMS messages are not importing.

Thank you for reporting this. Do your message addresses by any chance contain spaces? There's currently [a bug](https://github.com/tmo1/sms-ie/issues/159#issuecomment-2169712551) that prevents such messages from being imported. In any event, please provide a [minimal reproducible example](https://github.com/tmo1/sms-ie#minimal-reproducible-example).

**Comment by Savasdotexe** [comment]
Hi tmo1,

So first I tried to export then wipe and import 2 messages I had on the target device (Samsung A21s) yet still was unable to import anything. I tried the debugging option and selected 2000 and it imported successfully so I tried the same process with the large export I had it exported 1900, yet I only imported around 1700 so some messages are getting lost. If that helps.

I'll try and provide the logs in a few days, thank you.
